controllers/chat/moveToSign.js

```diff
@@ -25,10 +26,7 @@ const moveToSign = async (req, res) => {
 
   const client = StreamChat.getInstance(process.env.API_KEY, process.env.SECRET);
   try {
-    const [userModel, targetUserModel] = await Promise.all([
-      UsersFunction.findUserById(User, req?.userId),
-      UsersFunction.findUserById(User, targetUserId)
-    ]);
+    const [userModel] = await Promise.all([UsersFunction.findUserById(User, req?.userId)]);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of `targetUserModel` from the Promise.all might affect the subsequent code if it relies on this variable. Make sure to check the rest of the code for any references to `targetUserModel`.
```diff
-    const [userModel, targetUserModel] = await Promise.all([
-      UsersFunction.findUserById(User, req?.userId),
-      UsersFunction.findUserById(User, targetUserId)
-    ]);
+    const [userModel] = await Promise.all([UsersFunction.findUserById(User, req?.userId)]);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->